machineLearning/sentimentAnalysis/twitterSentimentAnalysis/twitterSentimentLambdaFunction/README.md

```diff
@@ -0,0 +1,113 @@
+# twitterSentimentLambdaFunction
+
+This project contains source code and supporting files for a serverless application that you can deploy with the SAM CLI. It includes the following files and folders.
+
+- hello_world - Code for the application's Lambda function and Project Dockerfile.
```

Comment:
Just with this readme can delete all this crap about deploying "Hello World" function.